cmd/client/main/main.go

```diff
@@ -182,10 +182,7 @@ func ContractMessageFromInput(value string, recipient string) ([]byte, error) {
 	}
 
 	// case recipBytes != 32
-	recipBytes, err := hex.DecodeString(recipient)
-	if err != nil {
-		return nil, errors.New("Failed to hex decode recipient")
-	}
```

Comment:
I added this change so that the contracts and update would work

Reply:
![image](https://user-images.githubusercontent.com/42977312/59990692-c4ac2980-9609-11e9-8d44-2b37e83a9461.png)
![image](https://user-images.githubusercontent.com/42977312/59990701-d2fa4580-9609-11e9-8774-1e61fe48ee30.png)
Oh, I thought since the test converts `recipientPKH` to a string with `string(..)` we should do the opposite in `ContractMessageFromInput(..)`.

Reply:
I'm not sure about the test but when I did the hex-decode in the function then the actual account info request and contract requests worked.

Reply:
Hmm.. when a recipient string is passed into the func, is that string supposed to be hex encoded? Or is it just the result of a string conversion from a byte slice?

Reply:
It is the result of `PrintInfo` in the client's `main.go` function, which prints out a hex-encoded version of the wallet address. So the input into this function should be a hex-encoded string. So we need to hex-decode it to get the right bytes.

Reply:
We should probably add some quality of life improvements in the future to ensure the user is using the correct address. Maybe spellchecking it to make sure the input string is length 64.

Reply:
Yea, that would be nice. I guess the `recipient` strings in `TestContractMessageFromInput(...)` need to be changed to be hex encoded as well.